includes/admin/class-sensei-onboarding.php

```diff
@@ -19,6 +19,25 @@
  * @since   3.1.0
  */
 class Sensei_Onboarding {
+	const ONBOARDING_USER_DATA_OPTION_NAME = 'sensei-onboarding-data';
+
+	/**
+	 * Default value for onboarding user data.
+	 *
+	 * @var array
+	 */
+	private $onboarding_user_data_defaults = [
+		'features'      => [],
+		'purpose'       => [],
```

Comment:
Do you think it would make sense to add the `purpose_other` inner the `purpose`, making it like an object where would have the `other` and the `selected`, as we'll use through the API?

Reply:
Yeah, it makes things a bit cleaner. 8e56fd07